include/xrpl/protocol/SField.h

```diff
@@ -68,6 +68,7 @@ class STCurrency;
     STYPE(STI_UINT128, 4)                         \
     STYPE(STI_UINT256, 5)                         \
     STYPE(STI_AMOUNT, 6)                          \
+    STYPE(STI_EITHER_AMOUNT, 6)                   \
```

Comment:
I see this duplicates st amount, and I think I know why it's done. Could we remove STI_AMOUNT?